terraform/prod.tfvars

```diff
@@ -52,6 +52,9 @@ tenant_cidrs = [
   "194.73.212.3/32",
   # MOJ Digital and Technology
   "81.134.202.29/32",
+  # UK Cloud / Skyscape egress for #140308141
+  185.40.8.212/32,
+  86.188.177.234/32,
```

Comment:
Actually, having thought about it for a minute, I think we should remove the provider from the description and just reference the story ID/URL.